src/linker/Linker/Driver.cs

```diff
@@ -998,6 +1032,7 @@ static void Usage ()
 			Console.WriteLine ("  -out PATH           Specify the output directory. Defaults to 'output'");
 			Console.WriteLine ("  --about             About the {0}", _linker);
 			Console.WriteLine ("  --verbose           Log messages indicating progress and warnings");
+			Console.WriteLine ("  --nowarn WARN-LIST  Disable specific warning messages.");
```

Comment:
redundant .

Reply:
This is the same message you get from `csc`.